frontend/src/views/ProjectView.vue

```diff
@@ -0,0 +1,88 @@
+<template>
+  <ul>
```

Comment:
I don't get It here. The upload part of `ProjectView` is designed differently from the `DragToStartField`.

Reply:
Yes, you need a different component because it looks different but it'd work similarly. I tried with `DragToStartField` and it works fine. You can add an un-designed version to this PR or leave it for the next one.